text-embeddings-inference/custom_server/config.yaml

```diff
@@ -0,0 +1,29 @@
+base_image:
+  # select an image: L4
+  # CPU	michaelf34/text-embeddings-inference:cpu-1.5
+  # Turing (T4, ...)	michaelf34/text-embeddings-inference:turing-1.5
+  # Ampere 80 (A100, A30)	michaelf34/text-embeddings-inference:1.5
+  # Ampere 86 (A10, A10G, A40, ...)	michaelf34/text-embeddings-inference:86-1.5
+  # Ada Lovelace (L4, ...)	michaelf34/text-embeddings-inference:89-1.5
+  # Hopper (H100/H100 40GB)	michaelf34/text-embeddings-inference:hopper-1.5
+  image: michaelf34/text-embeddings-inference:1.5
+model_metadata:
+  repo_id: BAAI/bge-base-en-v1.5
+docker_server:
+  start_command: sh -c "text-embeddings-router --port 7998 --model-id /data/local-model --max-client-batch-size 256 --max-concurrent-requests 40 --max-batch-tokens 32768"
```

Comment:
any guidance on these numbers? Maybe something we can call out in the README for how to set them

Reply:
Wrote a short essay in the readme!